docs/css/main.css

```diff
@@ -81,8 +81,7 @@ mark {
 }
 
 .site-nav-top {
-    margin: 0.8rem 0;
-    padding: 0 12px 12px 12px;
+    padding: 0.7rem 0 0.5rem 0.8rem;
```

Comment:
This makes this PR a breaking change, because you don't just need to fix it here, but also in https://github.com/MarkBind/markbind/blob/21620cb2d527ba7cb33a4d569ce442b83194f9c2/packages/core/template/default/stylesheets/main.css
and I believe all the test sites will need this update. Is this change very obvious? I tried undoing this and somehow the title looks about the same? If we can omit this then let's revert this change.

Reply:
Alright! The change was mainly to reduce the spacing below the `site-nav-top`. However, I think we can omit this change if this change makes the PR a breaking change.